### PR TITLE
Update readme with react-native example

### LINKED ISSUE #75: Coinbase pay sdk doesn't work on React Native
Hola guys,

When trying to integrate Coinbase pay with React Native, i'm getting an error when trying to `initOnRamp`:
`TypeError: window.addEventListener is not a function. (In 'window.addEventListener("message", onMessage)', 'window.addEventListener' is undefined)`

I tried to `generateOnRampURL` as well which threw this error:
`TypeError: Attempted to assign to readonly property.`

React native version being used: 0.63.5
Coinbase pay SDK version being used: 1.6.0

I followed the steps given [here](https://docs.cloud.coinbase.com/pay-sdk/docs/installing-sdk) to install the package

Is the sdk supported for React native (or mobile devices?)?
If yes, can you please help out with this error? If not, can you please point to the correct direction on how to use the sdk for mobile?

### PR DESCRIPTION
**Pull request by Perronef5**
### Description & motivation

Updating README to include example that will work with `ReactNative`

Closes #75

### CODE REVIEW

**Comment by Perronef5** [comment]
> ### Description & motivation
> Updating README to include example that will work with `ReactNative`

@lachiet Could I get some eyes on this? 👀

**Comment by Perronef5** [comment]
@zack-winchell-cb @lachiet ??

File: README.md [hunk 1]
```diff
@@ -126,6 +126,50 @@ const PayWithCoinbaseButton: React.FC = () => {
 };
 ```
 
+## React-Native example
+
+### Prerequisites
+
+``` 
+yarn add react-native-url-polyfill/auto
```
Comment (victor-magalhaes-cb) [inline README.md:134]:
```suggestion
yarn add react-native-url-polyfill
```

**Review by victor-magalhaes-cb — COMMENTED** [review]
Other than the typo, looks good to me! 😄

File: README.md (hunk 1 above)
Comment (qodesmith) [reply README.md:134]: @Perronef5 Could you merge or rebase off of master & push a new commit so we could merge this? 🙏

Comment (Perronef5) [reply README.md:134]: Will do 👍🏽

Comment (Perronef5) [reply README.md]: Done

**Review by vhfmag — APPROVED** [review]

**Review by qodesmith — APPROVED** [review]

**Comment by Perronef5** [comment]
@vhfmag @qodesmith I don't have the permission to merge the branch

**Pull request merged**